src/lib/di.container.ts

```
@@ -0,0 +1,59 @@
+
+  public static initialize() {
+    // Bind Prisma instance
+    this._container.bind(DITypes.Prisma).toConstantValue(new PrismaClient());
+
+    // Bind Clerk instance
+    this._container
+      .bind(DITypes.Clerk)
+      .toResolvedValue(async () => await clerkClient());
+
+    // Bind Stripe instance
+    this._container.bind(DITypes.Stripe).toConstantValue(
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)